common/src/main/java/com/wynntils/mc/mixin/DownloadedPackSourceMixin.java

```diff
@@ -32,4 +33,14 @@ private void onClearServerPackPre(CallbackInfoReturnable<CompletableFuture<Void>
             cir.cancel();
         }
     }
+
+    @Override
+    public void setServerPack(Pack pack) {
+        serverPack = pack;
+    }
+
+    @Override
+    public Pack getServerPack() {
+        return serverPack;
+    }
```

Comment:
In any other place we would have used an AW to make this public and write/read the field. I am not sure what is different here.

Reply:
Yeah, good point. I went through like 3-6 different solutions on how to achieve what I wanted, so when in the end I found a working solution, I did not think this through fully.